python/lsst/analysis/drp/colorColorFitPlot.py

```diff
@@ -229,152 +292,188 @@ def colorColorFitPlot(self, catPlot, plotInfo, fitParams):
+            ysFitLineHW = np.array([yMin, yMax])
+            xsFitLineHW = (ysFitLineHW - bHW)/mHW
+            ysFitLine = np.array([yMin, yMax])
+            xsFitLine = (ysFitLine - fitParams["bODR"])/fitParams["mODR"]
         else:
-            xsFitLineHW = np.array([fitParams["xMin"], fitParams["xMax"]])
-            ysFitLineHW = fitParams["mHW"]*xsFitLineHW + fitParams["bHW"]
-            xsFitLine = [fitParams["xMin"], fitParams["xMax"]]
+            xsFitLineHW = np.array([xMin, xMax])
+            ysFitLineHW = mHW*xsFitLineHW + bHW
+            xsFitLine = [xMin, xMax]
             ysFitLine = [fitParams["mODR"]*xsFitLine[0] + fitParams["bODR"],
                          fitParams["mODR"]*xsFitLine[1] + fitParams["bODR"]]
-            xsFitLine2 = [fitParams["xMin"], fitParams["xMax"]]
-            ysFitLine2 = [fitParams["mODR2"]*xsFitLine2[0] + fitParams["bODR2"],
-                          fitParams["mODR2"]*xsFitLine2[1] + fitParams["bODR2"]]
 
-        ax.plot(xsFitLineHW, ysFitLineHW, "w", lw=2)
+        ax.plot(xsFitLineHW, ysFitLineHW, "w", lw=1.5)
         lineHW, = ax.plot(xsFitLineHW, ysFitLineHW, "g", lw=1, ls="--", label="Hardwired")
+        ax.plot(xsFitLine, ysFitLine, "w", lw=1.5)
+        lineRefit, = ax.plot(xsFitLine, ysFitLine, "k", lw=1, ls="--", label="ODR Fit")
+        ax.legend(handles=[lineHW, lineRefit], fontsize=6, loc="lower right")
 
-        ax.plot(xsFitLine, ysFitLine, "w", lw=2)
-        lineInit, = ax.plot(xsFitLine, ysFitLine, "b", lw=1, ls="--", label="Initial")
-
-        ax.plot(xsFitLine2, ysFitLine2, "w", lw=2)
-        lineRefit, = ax.plot(xsFitLine2, ysFitLine2, "k", lw=1, ls="--", label="Refit")
-
-        # Calculate the distances to that line
-        # Need two points to characterise the lines we want
-        # to get the distances to
+        # Calculate the distances to that line for the data used in the fit.
+        # Two points are needed to characterise the lines we want to get the
+        # distances to.
         p1 = np.array([xsFitLine[0], ysFitLine[0]])
         p2 = np.array([xsFitLine[1], ysFitLine[1]])
 
-        p1HW = np.array([xsFitLine[0], ysFitLineHW[0]])
-        p2HW = np.array([xsFitLine[1], ysFitLineHW[1]])
+        p1HW = np.array([xsFitLineHW[0], ysFitLineHW[0]])
+        p2HW = np.array([xsFitLineHW[1], ysFitLineHW[1]])
 
         distsHW = perpDistance(p1HW, p2HW, zip(xs[fitPoints], ys[fitPoints]))
         dists = perpDistance(p1, p2, zip(xs[fitPoints], ys[fitPoints]))
 
         # Now we have the information for the perpendicular line we
         # can use it to calculate the points at the ends of the
         # perpendicular lines that intersect at the box edges
-        if np.fabs(fitParams["mHW"]) > 1:
-            xMid = (fitParams["yMin"] - fitParams["bODR2"])/fitParams["mODR2"]
+        if np.fabs(mHW) > 1:
+            xMid = (yMin - fitParams["bODR"])/fitParams["mODR"]
             xs = np.array([xMid - 0.5, xMid, xMid + 0.5])
```

Comment:
Should this be ODR rather than ODR2?

Reply:
Now I see the renaming in the other file, thanks git for making them in that order.